### PR TITLE
Expense Tracker (#309)

### LINKED ISSUE #309: Expense tracker
**Is your feature request related to a problem? Please describe.**
An Expense tracker will be created in react js

**Describe the solution you'd like**
The user can add the amount added or deducted and it will be stored in it.

### PR DESCRIPTION
**Pull request by Debashish-hub**
Fixes #309 

### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Description:

I have added expense tracker in react fixing the issue #309  

### Snapshot:

Provided in the readme of Expense tracker

### CODE REVIEW

**Comment by Tejas1510** [comment]
Good Work @Debashish-hub 
Just add the readme in the following format #145

**Comment by Debashish-hub** [comment]
Ok

**Comment by Debashish-hub** [comment]
I have updated the readme. Please check it @Tejas1510

**Comment by Debashish-hub** [comment]
I have added Typing Speed Test in Python. Please check it @Tejas1510

File: Python/Typing Speed Test/typingTest.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+import pygame
```
Comment (thegautamkumarjaiswal) [inline Python/Typing Speed Test/typingTest.py:1]: You can also comment little bit about work of package you used in the script, that will help to understand the package work.

File: Python/Typing Speed Test/typingTest.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            self.time_img = pygame.transform.scale(self.time_img, (150, 150))
+
+            #screen.blit(self.time_img, (80,320))
+            screen.blit(self.time_img, (self.w/2-75, self.h-140))
+            self.draw_text(screen, "Reset", self.h - 70, 26, (109, 248, 252))
+
+            print(self.results)
+            pygame.display.update()
+
+    def run(self):
```
Comment (thegautamkumarjaiswal) [inline Python/Typing Speed Test/typingTest.py:83]: Use comment to define work of functions

File: Reactjs/Expense Tracker/public/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```
Comment (thegautamkumarjaiswal) [inline Reactjs/Expense Tracker/public/index.html:1]: All work good! great

**Review by thegautamkumarjaiswal — APPROVED** [review]
Yes, I want to approve the work. Please merge this PR.

**Comment by GowthamGoush** [comment]
@Debashish-hub, could you please raise pull requests separately for different issues.

**Comment by Debashish-hub** [comment]
Added only Expense Tracker. Please merge it so that I can add another one @GowthamGoush

**Pull request merged**